src/attacks/periodic_attack_component.cpp

```diff
@@ -39,6 +39,9 @@ namespace ros_sec_test
 namespace attacks
 {
 
+PeriodicAttackComponent::PeriodicAttackComponent()
+: PeriodicAttackComponent("attack_node") {}
```

Comment:
This API is dangerous as we know we'll have more than one node in the process and having multiple nodes with the same name is undesirable. Let's remove this.